### PR TITLE
scripts: remove restriction for tgl rimage signing

### PR DESCRIPTION
**Pull request by juimonen**
Currently xtensa-build-all.sh has restriction to use rimage for tgl
signing, because rimage didn't support tgl. As rimage now has support
for tgl signing let's remove this limitation.

Signed-off-by: Jaska Uimonen <jaska.uimonen@intel.com>

### CODE REVIEW

**Comment by juimonen** [comment]
do not merge until https://github.com/thesofproject/rimage/pull/33 is merged, thanks

**Comment by marc-hb** [comment]
> do not merge until thesofproject/rimage#33 is merged, thanks

I made this a draft. Drafts cannot be merged.

Cc: @mbolivar

Very interesting how git fetch automatically and recursively fetches the git submodule change even when the corresponding PR hasn't been merged. Extremely useful for testing in CI. @juimonen was all that on purpose?

However I suspect this worked only because this submodule change is a request to pull a branch _in the central submodule repo itself_: it is not a request to pull a branch from a _forked_ submodule repo. Otherwise it would not find the submodule change. In other words not every submitter can test submodule changes.

@plbossart this means testing some kconfig changes would be trivial if kconfig were a submodule of the linux repo. I know how much you love submodules :-)

`[FEATURE] drop git submodules and switch to Zephyr's "west" multi-repo tool` #3517

**Comment by marc-hb** [comment]
There is no QEMU test in https://travis-ci.org/github/thesofproject/sof/builds/743350214, only a (successful) build test.

This should be a very small .travis.yml change assuming QEMU can work for TGL.

**Comment by marc-hb** [comment]
I could not resist testing this and wow! I stand corrected. 

> do not merge until thesofproject/rimage#33 is merged, thanks

It would be very confusing to merge this before rimage 33 is merged but it would interestingly not be a problem at all.  See gory details why in test PR #3609

tl;dr: git fetch can recursively fetch any random submodule SHA1 from any random github location. Nothing needs to be merged anywhere.

**Comment by lgirdwood** [comment]
@juimonen https://github.com/thesofproject/rimage/pull/33 is now merged.

**Review by singalsu — APPROVED** [review]
Nice. Though the acronyms like TGL in the commit message should be written in capitals.

**Review by ranj063 — APPROVED** [review]

**Review by lgirdwood — APPROVED** [review]
@juimonen can you change this to a normal PR.

**Comment by juimonen** [comment]
@lgirdwood now hopefully changed...

**Comment by marc-hb** [comment]
> Very interesting how git fetch automatically and recursively fetches the git submodule change even when the corresponding PR hasn't been merged. Extremely useful for testing in CI. @juimonen was all that on purpose?

Now it's officially documented: https://github.com/thesofproject/rimage/pull/35

**Pull request merged**
